totes

```diff
@@ -9,6 +9,15 @@ get_render_command() {
   fi
 }
 
+get_notes_path() {
+  echo "${XDG_DATA_HOME:-$HOME/.local/share}/totes"
+}
+
+get_suppressed_files_registry_spec() {
```

Comment:
what does the `spec` mean?

Reply:
comes from "filespec": https://en.wikipedia.org/wiki/Filespec

Reply:
meant to communicate "this is the full path, and not just the individual file name within the directory"

Reply:
and also as used twice in this line you added in an earlier commit: https://github.com/nathanshelly/totes/blob/44880f4e83bc0150423a26441b7d2006ff5cf168/totes#L26